README.md

```diff
@@ -105,6 +105,15 @@ Add this plugin to your pom.xml:
 To run diktat in **only-check** mode use command `$ mvn diktat:check@diktat`.
 To run diktat in **autocorrect** mode use command `$ mvn diktat:fix@diktat`.
 
+### Run with Maven using Spotless
```

Comment:
I think we need to unify these chapters.
You have duplication in Readme

Reply:
I thought that vice versa, it would be more clear if we have structure like `maven plugin -> spotless + maven` and then `gradle-plugin -> spotless + gradle`

Reply:
I do not think that it is a use-case

Reply:
Spotless is bonus. So it should be in a separate block.

Also do you really want text duplication in readme? I do not like it

Reply:
kk